src/main/resources/static/templates/PasswordResetVulnerability/Common/PasswordReset.css

```
@@ -0,0 +1,179 @@
+}
+
+.pr-detail-key {
+  font-weight: 600;
+  color: #444;
+}
+
+.pr-detail-value {
+  color: #1f2937;
+  text-align: right;
+  max-width: 65%;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)